README.md

```diff
@@ -13,8 +13,6 @@ The following database Docker image tags are currently supported:
 
 To add support for a new OS platform/version or database type/version, simply add a new Compose project like those under `projects` and point the scripts at it.
 
-NOTE: iRODS 4.2.x is the only supported series for testing at this time. 4.3.x and others will be added in the future.
```

Comment:
Is this PR meant to be for 4-2-stable or main?

Reply:
redirected the PR, this is moot.